projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../../styles/mixins';
+
+.lg-hero-img-card {
+  background-color: var(--hero-bg-color);
```

Comment:
As this is a separate component, I'm wondering if we need separate CSS vars? 🤔  E.g. `--hero-img-bg-color`. What do you think? 

CC @elenagarrone

Reply:
Yes, I think it would be better to have separate CSS variables 👍  
@IdaHW maybe `--hero-img-bg-color` in a new scss module.